presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -140,14 +140,24 @@ public BaseJdbcClient(JdbcConnectorId connectorId, BaseJdbcConfig config, String
     }
 
     @Override
-    public List<SchemaTableName> getTableNames(@Nullable String schema)
+    public List<SchemaTableName> getTableNames(@Nullable String schema, String tableNamePrefix)
```

Comment:
The `tableNamePrefix` parameter isn't used (except for a test). The change in `listTableColumns()` should be all that is needed.